### PR TITLE
Don't generate PDF receipt for skipped payments.

### PR DESCRIPTION
**Pull request by bjorntore**
Don't generate PDF receipt for skipped payments.

## Related Issue(s)
- #711 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.App.Core/Internal/Process/ProcessTasks/PaymentProcessTask.cs [hunk 1]
```diff
@@ -52,7 +53,12 @@ public async Task End(string taskId, Instance instance)
     {
         AltinnPaymentConfiguration paymentConfiguration = GetAltinnPaymentConfiguration(taskId);
 
-        if (!await _paymentService.IsPaymentCompleted(instance, paymentConfiguration.Validate()))
+        PaymentStatus paymentStatus = await _paymentService.GetPaymentStatus(instance, paymentConfiguration.Validate());
```
Comment (bjorntore) [inline src/Altinn.App.Core/Internal/Process/ProcessTasks/PaymentProcessTask.cs:56]: This is the real change. Don't create PDF if status is skipped.

**Review by cammiida — APPROVED** [review]
Nå har jeg ikke alt av kontekst og sånt, men LGTM 🙌

**Review by Magnusrm — APPROVED** [review]
LGTMT! 😁

**Review by danielskovli — APPROVED** [review]
LGTM 👍

**Pull request merged**